Makefile

```diff
@@ -450,13 +450,13 @@ update-dependencies: ## Uses go get -u to update all the dependencies while hold
 
 vet: ## Run mattermost go vet specific checks
 	@if ! [ -x "$$(command -v mattermost-govet)" ]; then \
-		echo "mattermost-govet is not installed. Please install it executing \"GO111MODULE=off go get -u github.com/mattermost/mattermost-govet\""; \
+		echo "mattermost-govet is not installed. Please install it executing \"GO111MODULE=off GOBIN=$(PWD)/bin go get -u github.com/mattermost/mattermost-govet\""; \
 		exit 1; \
 	fi; \
 
-	$(GO) vet -vettool=$(GOPATH)/bin/mattermost-govet -license -structuredLogging ./...
+	$(GO) vet -vettool=$(GOBIN)/mattermost-govet -license -structuredLogging ./...
```

Comment:
That makes it hard for someone that is developing `mattermost-govet` to run his/her local changes with the server repo. Maybe we stay with default `GOBIN` at `$HOME/go/bin` for this specific binary?

Reply:
One can easily do that by setting a custom GOBIN path while installing the mmgovet binary, or set the right path for `-o` output. I think we should keep the development of mmgovet and running mmgovet 2 separate concerns. And given that the problem can be solved pretty easily, I would suggest keeping all tools inside the project directory.